packit_service/worker/checker/distgit.py

```diff
@@ -164,6 +164,18 @@ def pre_check(self) -> bool:
         return self.project_to_sync is not None
 
 
+class TaggedBuildIsNotABuildOfSelf(Checker):
```

Comment:
Can you please add a short docstring to make it really clear also for not-so-experienced packagers? Thanks!

Reply:
I'm not sure this is related to packaging experience, this is to break the cycle - completing the build generates an event that triggred the build in the first place, we don't want to repeat that indefinitely (though in this case it would fail anyway because Koji doesn't allow to build the same NVR twice).

Reply:
@lachmanfrantisek WDYT?